### DISCUSSION TITLE
Specify sort order for Workflows

### DISCUSSION THREAD

**Discussion by elliot-nelson** [community/community#110799] [Actions]
### Select Topic Area

Product Feedback

### Body

I would love to be able to change the _display order_ of workflows in the Actions tab using YAML properties in the workflow.

For example, a property like `display_order: 17`.

This new property could follow a few simple rules:

 - Workflows with no `display_order` come below all workflows with a `display_order`
 - Workflows with the same `display_order` are sorted alphabetically as they are today
 - Workflows with `display_order: -1` are _hidden_ in the list (they show up only by direct link) -- this is optional but it would definitely sweeten the pot to be able to cut your reusable workflows right out of the list altogether.

**Comment by ccharlton** [comment]
+1
